xml/System.Security.Cryptography.X509Certificates/CertificateRequest.xml

```diff
@@ -267,7 +353,43 @@
         <param name="signatureGenerator">To be added.</param>
         <summary>To be added.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+              When submitting a certificate signing request via a web browser, or other graphical or textual
+              interface, the input is frequently expected to be in the PEM (Privacy Enhanced Mail) format,
+              instead of the DER binary format. To convert the return value to PEM format, make a string
+              consisting of `-----BEGIN CERTIFICATE REQUEST-----`, a newline, the Base-64-encoded
+              representation of the request (by convention, linewrapped at 64 characters), a newline,
+              and `-----END CERTIFICATE REQUEST-----`.
+            
+              ```csharp
+              public static string PemEncodeSigningRequest(CertificateRequest request, PkcsSignatureGenerator generator)
+              {
+                  byte[] pkcs10 = request.CreateSigningRequest(generator);
+                  StringBuilder builder = new StringBuilder();
+                
+                  builder.AppendLine("-----BEGIN CERTIFICATE REQUEST-----");
+                
+                  string base64 = Convert.ToBase64String(pkcs10);
+                
+                  int offset = 0;
+                  const int LineLength = 64;
+                
+                  while (offset < base64.Length)
+                  {
+                      int lineEnd = Math.Min(offset + LineLength, base64.Length);
+                      builder.AppendLine(base64.Substring(offset, lineEnd - offset));
+                      offset = lineEnd;
+                  }
+                
+                  builder.AppendLine("-----END CERTIFICATE REQUEST-----");
+                  return builder.ToString();
+                }
+                ```
```

Comment:
I'm happy for this to go in the documentation (that's why I wrote it, after all).  But if it doesn't match anything we do anywhere else I'm also content to have it cut.